stairway/src/main/java/bio/terra/stairway/FlightFilter.java

```diff
@@ -177,6 +213,35 @@ public FlightFilter submittedTimeSortDirection(
     return this;
   }
 
+  /**
+   * Create an input parameter filter object. This is processed by converting the {@code value} into
+   * JSON and doing a string comparison against the input parameter stored in the database. The
+   * {@code value} object must be <b>exactly</b> the same class as the input parameter.
+   *
+   * @param key name of the parameter to compare
+   * @param op a {@link FlightFilterOp}
+   * @param value some object for comparison
+   * @return A newly created FlightFilterPredicate object
+   * @throws FlightFilterException if predicate is not supplied
+   */
+  public FlightFilterPredicate makeInputPredicate(String key, FlightFilterOp op, Object value) {
+    if (key == null) {
+      throw new FlightFilterException("Key must be specified in an input filter");
+    }
+    if (value == null && op != FlightFilterOp.EQUAL) {
+      throw new FlightFilterException(
+          "Value cannot be null in an input filter if not doing an equality check");
+    }
+
+    if (op == FlightFilterOp.IN) {
+      return new FlightFilterPredicate(key, op, value, Datatype.LIST, makeParameterName());
+    } else if (value == null) {
```

Comment:
I'm wondering why you chose to do this, rather than an explicit op like `FlightFilterOp.IS_NULL`. Also, is there a way to check `NOT NULL`?

Reply:
There was already logic for NULL datatypes so it seemed more appropriate to keep that